src/crystal/system/win32/random.cr

```diff
@@ -1,9 +1,15 @@
+require "c/ntsecapi"
+
 module Crystal::System::Random
   def self.random_bytes(buf : Bytes) : Nil
-    raise NotImplementedError.new("Crystal::System::Random.random_bytes")
+    unless LibC.RtlGenRandom(buf, buf.size)
```

Comment:
Will this work even though it can't be nil?

Reply:
huh?

Reply:
Oops, fixed.